src/lib/stream/rc4/rc4.cpp

```diff
@@ -111,6 +111,6 @@ void RC4::clear()
 /*
 * RC4 Constructor
 */
-RC4::RC4(size_t s) : SKIP(s), X(0), Y(0) {}
+RC4::RC4(size_t s) : SKIP(s), X(0), Y(0), state(), buffer(), position(0) {}
```

Comment:
I think `state` and `buffer` are default initialized (i.e. empty vector) anyway, are they really causing warnings? Can we move `byte X = 0; byte Y = 0; size_t position = 0` to the header? Those initializations are not specific for this constructor but useful default.